GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -218,7 +219,12 @@ namespace gdjs {
               // Clear the depth as each layer is independent and display on top of the previous one,
               // even 3D objects.
               threeRenderer.clearDepth();
-              threeRenderer.render(threeScene, threeCamera);
+              if (runtimeLayerRenderer.hasPostProcessingPass()) {
+                threeEffectComposer.render();
+              } else {
+                threeRenderer.render(threeScene, threeCamera);
+              }
```

Comment:
The switch is done dynamically because it's easier to detect. Otherwise, we would have to declare which Effect is a post-processing effect of Three.js.